afterSignHook.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { notarize } = require('electron-notarize')
+
+/**
+ *
+ * @param {*} context - Object containint electron-builder context
+ * @returns {Promise}
+ *
+ * @example
+ * await notarize({
+ *   // electron-builer context
+ * })
+ */
+const notarizing = async (context) => {
+  const { electronPlatformName, appOutDir } = context
+  if (electronPlatformName !== 'darwin') {
+    return Promise.resolve()
+  }
+
+  const appName = context.packager.appInfo.productFilename
+  const appleId = 'accounts+apple@balena.io'
+
+  return notarize({
```

Comment:
`await` would make more sense than `return` here.

Reply:
IIRC returning a Promise in an async function is the same as `return await`ing a Promise

Reply:
`await` expresses the intent, which is to wait for `notarize` to finish. It doesn't return anything anyway.

Reply:
Also it makes clear why this function is `async`. It shouldn't be async if there's no `await` in it.